superset/sql_parse.py

```diff
@@ -26,6 +27,13 @@ def tables(self):
     def is_select(self):
         return self.sql.upper().startswith('SELECT')
 
+    def stripped(self):
+        sql = self.sql
+        if sql:
+            while sql[-1] in (' ', ';', '\n', '\t'):
```

Comment:
those are the only blank characters I could think of, there'll be this place to add new ones if it comes up